README.md

```diff
@@ -441,12 +441,12 @@ The registration of the service account is handled automatically during ``make
 deploy``. To register  without deploying, run ``make -C terraform sam``. The
 authorization must be performed manually by a developer with administrator access to
 the SAM group that controls access to TDR. Currently that group is called
-`JadeStewards-dev`. *Jade* is the Broad-internal codename for TDR.
+`azul-dev`.
 
-A member of the *Jade stewards* group has read and write access to TDR, and an
+A member of the *azul-dev* group has read and write access to TDR, and an
```

Comment:
Are we sure that `azul-dev` grants write access? I know `JadeStewards-dev` did but I thought part of the reason for moving away from it was because we didn't need write access. I could be completely wrong, just something to consider double-checking.

Reply:
`azul-dev` is read only 👍🏽